src/Twig/RichEditorExtension.php

```diff
@@ -250,7 +251,7 @@ public function getDefaultElementDataField(): string
      *
      * @return string|null
      */
-    public function getCurrentFilePath(array $context): ?string
+    public function getCurrentFilePath(array $context, string $varName): ?string
```

Comment:
Is this new param wouldn't introduce bugs for people who ever have the plugin installed and use the `monsieur biz_richeditor_get_current file_path ` twig function for any reason? Default value maybe?